test/tests/api/v1_1/nodes_tests.py

```diff
@@ -92,21 +94,28 @@ def __post_workflow(self, id, graph_name, data):
             sleep(1)
             timeout -= 1
         return timeout
-
-    def check_compute_count(self):
+    
+    def get_undiscovered_compute_count(self):
         Nodes().nodes_get()
         nodes = self.__get_data()
-        count = 0
+        discovered_count = 0
         for n in nodes:
             type = n.get('type')
             if type == 'compute':
-                count += 1
-        return count
+                id = n.get('id')
+                status = self.__get_workflow_status(id)
+                if status != 'pending' and status != 'running':
```

Comment:
Thanks, I got it. 

So in V1_1 and V2_0 tests, instead of "find in db and no related active workflow", we should check catalogs and make sure a node is actually discovered.

"to_be_discovered" is still useful, right?